### DISCUSSION TITLE
【0x07】Token 自动售卖机 | Web3.0 dApp 开发（七）

### DISCUSSION THREAD

**Discussion by leeduckgo** [NonceGeek/Move-Camp#56] [Projects]
> https://mp.weixin.qq.com/s/FpAQlqWoxXyVVKzDg9vHvQ

Video:

> //TBD

完成的可以留言部署此 dApp 的 `github-pages` 地址，打卡有惊喜。

**Comment by qiwihui** [comment]
我完成啦！

部署页面地址：http://qiwihui-scaffold-2.surge.sh/
项目代码地址：https://github.com/qiwihui/scaffold-eth-typescript-challenges/tree/challenge-2-token-vendor
之前写的教程地址：https://mirror.xyz/qiwihui.eth/Tkm4eLO7VnZi2X05vdIfTzcMvgrVQJfG97xd_wPnGQw

**Reply by leeduckgo** [reply]
Cool!~

**Comment by gofenix** [comment]
<img width="1611" alt="image" src="https://user-images.githubusercontent.com/6822558/159105423-6d03c8db-35ba-4a82-835d-919f2c890c05.png">

https://mysterious-toe.surge.sh/debug

**Comment by leeduckgo** [comment]
Well Done! Bro!
